sunny_osm/README.md

```diff
@@ -0,0 +1,92 @@
+# OSM and phtovoltaic data
+
+## Prerequisites
+
+Check the prerequisites [here](../README.md).
+
+## Introduction
+
+The OSM data is provided by [GeoFabrik's](https://download.geofabrik.de/europe/france/languedoc-roussillon.html) regional download, the French administrative data by the [IGN](https://geoservices.ign.fr/adminexpress) and the photovoltaic potential data by [Solargis](https://solargis.com/resources/free-maps-and-gis-data?locality=france).
+
+## Get in the right place
+
+```shell
+cd meteo_france
```

Comment:
`sunny_osm`, or even `buildings_solar_potential`, wouldn't be clearer ?

Reply:
Well that is just a mistake in my copy/paste, changing that !